### PR TITLE
JS compatibility

### PR DESCRIPTION
**Pull request by NicklasXYZ**
Solves issue https://github.com/gleam-lang/bitwise/issues/6 by making the lib compatible with the JS compilation target.
Various lib files have also been updated.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ jobs:
       - uses: erlef/setup-beam@v1.9.0
         with:
           otp-version: "23.2"
-          gleam-version: "0.20.0"
-      - run: gleam test
+          gleam-version: "0.21.0"
       - run: gleam format --check src test
+      - run: gleam deps download
+      - run: gleam test
```
Comment (lpil) [inline .github/workflows/test.yml:20]: Can we run the tests for both the JavaScript and the Erlang targets here please

File: README.md [hunk 2]
```diff
@@ -1,17 +1,24 @@
...
-# Run the eunit tests
-rebar3 eunit
+## Installation
 
-# Run the Erlang REPL
-rebar3 shell
+If available on Hex this package can be added to your Gleam project:
+
+```sh
+gleam add bitwise
```
Comment (lpil) [inline README.md:21]:
```suggestion
gleam add gleam_bitwise
```

File: gleam.toml [hunk 3]
```diff
@@ -10,7 +10,7 @@ links = [
 ]
 
 [dependencies]
-gleam_stdlib = "~> 0.18"
+gleam_stdlib = "~> 0.21"
 
 [dev-dependencies]
-gleeunit = "~> 0.1"
+gleeunit = "~> 0.6.1"
```
Comment (lpil) [inline gleam.toml:16]:
```suggestion
gleeunit = "~> 0.6"
```

File: manifest.toml [hunk 4]
```diff
@@ -2,10 +2,10 @@
...
-  { name = "gleeunit", version = "0.4.0", build_tools = ["gleam"], requirements = ["gleam_stdlib"], otp_app = "gleeunit", source = "hex", outer_checksum = "E8FDD4E31E0BBB87F625B0BB4DEABDE780180F1C2B15A4534442823066468638" },
+  { name = "gleam_stdlib", version = "0.21.0", build_tools = ["gleam"], requirements = [], otp_app = "gleam_stdlib", source = "hex", outer_checksum = "FB03CD50B477867DA65B1318252ABA02F76175754762D15BF6C792CF5B85BCC4" },
+  { name = "gleeunit", version = "0.6.1", build_tools = ["gleam"], requirements = ["gleam_stdlib"], otp_app = "gleeunit", source = "hex", outer_checksum = "5BF486C3E135B7F5ED8C054925FC48E5B2C79016A39F416FD8CF2E860520EE55" },
 ]
 
 [requirements]
-gleam_stdlib = "~> 0.18"
-gleeunit = "~> 0.1"
+gleam_stdlib = "~> 0.21"
+gleeunit = "~> 0.6.1"
```
Comment (lpil) [inline manifest.toml:11]:
```suggestion
gleeunit = "~> 0.6"
```

**Comment by lpil** [comment]
Wonderful thank you. Some small comments above

File: .github/workflows/test.yml (hunk 1 above)
Comment (NicklasXYZ) [reply .github/workflows/test.yml:20]: Sure, I've added the command :)

**Review by lpil — APPROVED** [review]
Thank you!

**Pull request merged**